src/CMakeLists.txt

```diff
@@ -137,6 +162,6 @@ if(MSVC)
     )
 else()
     target_compile_options(${PROJECT_NAME} PRIVATE
-        -Wall -Wextra -Werror
```

Comment:
It's better to fix the warnings

Reply:
The issue is that gconf library is extremely old and cause deprecated-declarations warnings. Added -Wno-deprecated-declarations for this particular file